govc/object/method.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package object
+
+import (
+	"context"
+	"flag"
+
+	"github.com/vmware/govmomi/govc/cli"
+	"github.com/vmware/govmomi/govc/flags"
+	"github.com/vmware/govmomi/object"
+)
+
+type method struct {
+	*flags.DatacenterFlag
+
+	name   string
+	reason string
+	source string
+	enable bool
+}
+
+func init() {
+	cli.Register("object.method", &method{})
+}
+
+func (cmd *method) Register(ctx context.Context, f *flag.FlagSet) {
+	cmd.DatacenterFlag, ctx = flags.NewDatacenterFlag(ctx)
+	cmd.DatacenterFlag.Register(ctx, f)
+
+	f.StringVar(&cmd.name, "name", "", "Method name")
+	f.StringVar(&cmd.reason, "reason", "", "Reason for disabling method")
+	f.StringVar(&cmd.source, "source", "govc", "Source ID")
+	f.BoolVar(&cmd.enable, "enable", true, "Enable method")
```

Comment:
Should this be `enabled` rather than `enable`? That would be consistent with the esxcli style but honestly don't recall for the rest of govc.